src/bitfield/models.py

```diff
@@ -148,4 +148,34 @@ def deconstruct(self):
         return name, path, args, kwargs
 
 
+class TypedBitfieldMeta(type):
```

Comment:
nice, I was building the same thing

Reply:
oh, i should've synced. should we have a slack channel for typing efforts?